src/Identity/UI/src/Microsoft.AspNetCore.Identity.UI.csproj

```diff
@@ -45,22 +34,4 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="_UpdatedIdentityUIStaticWebAssets">
-
-    <ItemGroup>
-      <StaticWebAsset Remove="@(StaticWebAsset)" />
-
-      <_V4Content Include="wwwroot\V4\**" />
-
-      <StaticWebAsset Include="@(_V4Content->'%(FullPath)')">
```

Comment:
Does this affect @HaoK 's ongoing work for bootstrap 5?

Reply:
I already spoke to him about it and we'll bring this back in RC1